src/bin/nydus-image/main.rs

```diff
@@ -773,8 +771,18 @@ fn main() -> Result<()> {
     } else if let Some(matches) = cmd.subcommand_matches("unpack") {
         Command::unpack(matches)
     } else {
-        println!("{}", usage);
-        Ok(())
+        #[cfg(target_os = "linux")]
+        if let Some(matches) = cmd.subcommand_matches("export") {
+            Command::export(&cmd, matches, &build_info)
+        } else {
+            println!("{}", usage);
+            Ok(())
+        }
+        #[cfg(not(target_os = "linux"))]
+        {
+            println!("{}", usage);
```

Comment:
Thanks for the fixup! Maybe give the warning to tell users we can't support `export` subcommand for macos here?

Reply:
Actually if the user run `export` on macos, it cannot be recognized.

```shell
% ./target/debug/nydus-image export
error: The subcommand 'export' wasn't recognized

Usage: nydus-image [OPTIONS] [COMMAND]

For more information try '--help'
```

IMHO this error msg is enough, isn't it ? :)

Reply:
Also `export` has been removed from the available subcmd on macos:

```shell
% ./target/debug/nydus-image -h
Build, analyze, inspect or validate RAFS filesystems/Nydus accelerated container images

Usage: nydus-image [OPTIONS] [COMMAND]

Commands:
  create     Create RAFS filesystems from directories, tar files or OCI images
  chunkdict  deduplicate RAFS filesystem metadata
  merge      Merge multiple bootstraps into a overlaid bootstrap
  check      Validate RAFS filesystem metadata
  inspect    Inspect RAFS filesystem metadata in interactive or request mode
  stat       Generate statistics information for RAFS filesystems
  compact    (experimental)Compact specific nydus image, remove unused chunks in blobs, merge small blobs
  unpack     Unpack a RAFS filesystem to a tar file
  help       Print this message or the help of the given subcommand(s)

Options:
  -L, --log-file <log-file>    Log file path
  -l, --log-level <log-level>  Log level: [default: info] [possible values: trace, debug, info, warn, error]
  -h, --help                   Print help information
  -V, --version                Print version information

```

Reply:
Okay, maybe some macos users ask why not have `export` subcommand on his mac, but I think it's also okay for me.